catroid/src/main/java/org/catrobat/catroid/web/ServerCalls.java

```diff
@@ -54,120 +54,49 @@
-	private static final String GET_FACEBOOK_USER_INFO_URL = FlavoredConstants.BASE_URL_HTTPS
-			+ "api/getFacebookUserInfo/getFacebookUserInfo.json";
-	private static final String CHECK_EMAIL_AVAILABLE_URL = FlavoredConstants.BASE_URL_HTTPS
-			+ "api/EMailAvailable/EMailAvailable.json";
-	private static final String CHECK_USERNAME_AVAILABLE_URL = FlavoredConstants.BASE_URL_HTTPS
-			+ "api/UsernameAvailable/UsernameAvailable.json";
-	private static final String EXCHANGE_GOOGLE_CODE_URL = FlavoredConstants.BASE_URL_HTTPS
-			+ "api/exchangeGoogleCode/exchangeGoogleCode.json";
-	private static final String EXCHANGE_FACEBOOK_TOKEN_URL = FlavoredConstants.BASE_URL_HTTPS
-			+ "api/exchangeFacebookToken/exchangeFacebookToken.json";
-	private static final String GOOGLE_LOGIN_URL = FlavoredConstants.BASE_URL_HTTPS
-			+ "api/loginWithGoogle/loginWithGoogle.json";
-	private static final String FACEBOOK_LOGIN_URL = FlavoredConstants.BASE_URL_HTTPS
-			+ "api/loginWithFacebook/loginWithFacebook.json";
-	private static final String TEST_CHECK_TOKEN_URL = BASE_URL_TEST_HTTPS + "api/checkToken/check.json";
-	private static final String TEST_LOGIN_URL = BASE_URL_TEST_HTTPS + "api/login/Login.json";
-	private static final String TEST_REGISTRATION_URL = BASE_URL_TEST_HTTPS + "api/register/Register.json";
-	private static final String TEST_CHECK_GOOGLE_TOKEN_URL = BASE_URL_TEST_HTTPS
-			+ "api/GoogleServerTokenAvailable/GoogleServerTokenAvailable.json";
-	private static final String TEST_CHECK_FACEBOOK_TOKEN_URL = BASE_URL_TEST_HTTPS
-			+ "api/FacebookServerTokenAvailable/FacebookServerTokenAvailable.json";
-	private static final String TEST_GET_FACEBOOK_USER_INFO_URL = BASE_URL_TEST_HTTPS
-			+ "api/getFacebookUserInfo/getFacebookUserInfo.json";
-	private static final String TEST_CHECK_EMAIL_AVAILABLE_URL = BASE_URL_TEST_HTTPS
-			+ "api/EMailAvailable/EMailAvailable.json";
-	private static final String TEST_CHECK_USERNAME_AVAILABLE_URL = BASE_URL_TEST_HTTPS
-			+ "api/UsernameAvailable/UsernameAvailable.json";
-	private static final String TEST_EXCHANGE_GOOGLE_CODE_URL = BASE_URL_TEST_HTTPS
-			+ "api/exchangeGoogleCode/exchangeGoogleCode.json";
-	private static final String TEST_EXCHANGE_FACEBOOK_TOKEN_URL = BASE_URL_TEST_HTTPS
-			+ "api/exchangeFacebookToken/exchangeFacebookToken.json";
-	private static final String TEST_GOOGLE_LOGIN_URL = BASE_URL_TEST_HTTPS
-			+ "api/loginWithGoogle/loginWithGoogle.json";
-	private static final String TEST_FACEBOOK_LOGIN_URL = BASE_URL_TEST_HTTPS
-			+ "api/loginWithFacebook/loginWithFacebook.json";
-	private static final String TEST_DELETE_TEST_USERS = BASE_URL_TEST_HTTPS
-			+ "api/deleteOAuthUserAccounts/deleteOAuthUserAccounts.json";
-	private static final String JSON_STATUS_CODE = "statusCode";
-	private static final String JSON_ANSWER = "answer";
-	private static final String JSON_TOKEN = "token";
-
-	private static final ServerCalls INSTANCE = new ServerCalls();
-
 	public static boolean useTestUrl = false;
```

Comment:
please remove this from `HttpRequestsKt` and reduce visibility (private or package private, if possible)

Reply:
removed from HttpRequestKt, but unfortunately it is not possible to reduce the visibility.